### PR TITLE
Contact form submits to discord

### PR DESCRIPTION
**Pull request by sarahschwartz**
we just need to add the webhook to the environment variables

### CODE REVIEW

File: pages/contact.js [hunk 1]
```diff
@@ -43,15 +60,16 @@ export default function Contact() {
         console.log("form submitted successfully !!!");
```
Comment (amoweolubusayo) [inline pages/contact.js:60]: Hey Sarah, weldone! Would we be needing the console.log here?

File: pages/contact.js [hunk 2]
```diff
@@ -43,15 +60,16 @@ export default function Contact() {
...
         //set a success banner here
       }
-      //check response, if success is false, dont take them to success page
+      // check response, if success is false, dont take them to success page
     } catch (error) {
       console.log("there was an error submitting", error);
```
Comment (amoweolubusayo) [inline pages/contact.js:65]: Hey Sarah, weldone! Would we be needing the console.log here?

File: pages/contact.js [hunk 3]
```diff
@@ -43,15 +60,16 @@ export default function Contact() {
...
     }
   };
 
   const resetForm = () => {
     setFirstName("");
-    setEmail("");
```
Comment (amoweolubusayo) [inline pages/contact.js:54]: Awesome

**Review by amoweolubusayo — APPROVED** [review]

File: pages/contact.js (hunk 1 above)
Comment (sarahschwartz) [reply pages/contact.js:60]: thanks! no we can take that out. I think we should probably add some confirmation on the page at some point too but I just didn't get there

**Pull request merged**